api/response/schema.py

```diff
@@ -13,6 +13,8 @@ class Response(SQLAlchemyObjectType):
     class Meta:
         model = ResponseModel
 
+    room_response_count = graphene.Int()
```

Comment:
is it room response or question response?

Reply:
There is already a question_response variable in the query as can be shown in the screenshot.

Reply:
I have updated it to question_response_count_in_room, @Bonifase.